fastify.js

```diff
@@ -355,7 +355,8 @@ function fastify (options) {
         if (typeof address === 'string') {
           return address
         }
-        return `${this[kOptions].https ? 'https' : 'http'}://${address.address}:${address.port}`
+        const host = address.address === '::1' ? '[::1]' : address.address
```

Comment:
Would it be better?
```suggestion
        const host = address.family === 'IPv6' ? `[${address.address}]` : address.address
```

Reply:
I'm not that familiar with URLs and IPv6 to know, will confirm

Reply:
The suggestion is correct. 

http://[::1]:8000/hello

Reply:
https://www.rfc-editor.org/rfc/rfc3986#section-1.1.2
https://www.rfc-editor.org/rfc/rfc2732#section-2